aws-observability/templates/test/TestTemplate.yaml

```diff
@@ -1410,21 +1418,21 @@ Tests:
             Section3aCloudTrailLogsSourceName: ''
             Section3bCloudTrailSourceUpdated: 'No'
           sumoSNSAppStack:
-            Section2aInstallApp: ‘Yes’
-            Section3aCloudTrailLogsSourceName: ‘*cloudtrail-logs*’
-            Section3bCloudTrailSourceUpdated: ‘No’
+            Section2aInstallApp: 'No'
+            Section3aCloudTrailLogsSourceName: ''
+            Section3bCloudTrailSourceUpdated: 'No'
           sumoSQSAppStack:
-            Section2aInstallApp: ‘Yes’
-            Section3aCloudTrailLogsSourceName: ‘*cloudtrail-logs*’
-            Section3bCloudTrailSourceUpdated: ‘No’
+            Section2aInstallApp: 'No'
```

Comment:
Why installapp is set to know ?

Reply:
we are not installing any app on this testcases.